src/BenchmarkDotNet/Running/BenchmarkSwitcher.cs

```diff
@@ -65,7 +64,7 @@ public IEnumerable<Summary> Run(string[] args = null, IConfig config = null)
         [MethodImpl(MethodImplOptions.NoInlining)]
         private IEnumerable<Summary> RunWithDirtyAssemblyResolveHelper(string[] args, IConfig config)
         {
-            var logger = config.GetNonNullCompositeLogger();
+            var logger = config.GetNonNullCompositeLogger();            
```

Comment:
nit: formatting: trailing whitespaces

```suggestion
            var logger = config.GetNonNullCompositeLogger();
```